about.hbs

```diff
@@ -1,7 +1,8 @@
-// Be careful to prevent auto-formatting from breaking this file's indentation
-// Replace this file with JSON output once this is resolved: https://github.com/EmbarkStudios/cargo-about/issues/73
-
-module.exports = [
+{{!
+Be careful to prevent auto-formatting from breaking this file's indentation
+Replace this file with JSON output once this is resolved: https://github.com/EmbarkStudios/cargo-about/issues/73
+}}
+GRAPHITE-LICENSES: [
```

Comment:
I'm confused what this does. What is `GRAPHITE-LICENSES: `? It's not a valid variable name since it includes a hyphen, and you're using a colon, and it doesn't include a declaration like `const`, but the file is also not wrapped in `{}` so it's not a JavaScript object?

Reply:
it's just a marker to make sure that cargo-about outputs something we expect, before we run the output as javascript. `cargo install cargo-about` always installs the latest version, which may not be backwards compatible with what we expect.

Probably overkill defensive programming, but it doesn't hurt. And eventually we'll replace it with JSON anyway.

Reply:
I see. I thought this was some weird JS syntax that I was unfamiliar with, but now I understand we preprocess this file's string to remove it. To avoid causing the confusion I faced, I'd suggest moving this to a line above and make it into actual valid syntax: a string statement.

```js
"This JS was generated by cargo-about and the presence of this text indicates the whole file, including the following license array expression, should be valid.";

[...]
```

(This is valid syntax because `eval("'Hello World'; [123, 456]");` returns `[123, 456]`.)

You can check for that string expression (I'd suggest [startsWith()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) rather than `substr()` for simplicity) and then you don't even have to bother using substring to remove it from what `eval()` sees.

Reply:
@cauthmann Sorry, I apparently marked this as resolved by accident. I just updated the branch on top of master so we're almost ready to merge, but if you could take a look at the above comment, that would be much appreciated. Sorry for the delay in the past few days, I've been out of town until now.